ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_node_type.schema.yml

```
@@ -199,6 +199,14 @@ $defs:
               table: node-type-uplink-configuration
             description: |
               Set point-to-Point interface speed and will apply to uplink interfaces on both ends.
+              (Uplink switch interface speed can be overridden with 'uplink_switch_interface_speed').
+              interface_speed or forced interface_speed or auto interface_speed.
```

I know this was there already but since we are touching it:
```suggestion
              Speed should be set in the format `<interface_speed>` or `forced <interface_speed>` or `auto <interface_speed>`.
```
And same below.